patches/devel.R-cran-callr.3.5.1.diff

```diff
@@ -0,0 +1,67 @@
+diff --git a/devel/Makefile b/devel/Makefile
+index c155877b05eb..4954fe191480 100644
+--- a/devel/Makefile
++++ b/devel/Makefile
+@@ -25,6 +25,7 @@
+     SUBDIR += R-cran-brio
+     SUBDIR += R-cran-broom
+     SUBDIR += R-cran-caTools
++    SUBDIR += R-cran-callr
+     SUBDIR += R-cran-caret
+     SUBDIR += R-cran-chron
+     SUBDIR += R-cran-classInt
+diff --git a/devel/R-cran-callr/Makefile b/devel/R-cran-callr/Makefile
+new file mode 100644
+index 000000000000..3c0f3d31dea2
+--- /dev/null
++++ b/devel/R-cran-callr/Makefile
+@@ -0,0 +1,29 @@
++# $FreeBSD$
++# Created by: Guangyuan Yang <ygy@FreeBSD.org>
++
++PORTNAME=	callr
++DISTVERSION=	3.5.1
++CATEGORIES=	devel
++DISTNAME=	${PORTNAME}_${DISTVERSION}
++
++MAINTAINER=	ygy@FreeBSD.org
++COMMENT=	Call R from R
++
++LICENSE=	MIT
++LICENSE_FILE=	${WRKSRC}/LICENSE
++
++RUN_DEPENDS=	R-cran-R6>0:devel/R-cran-R6 \
++		R-cran-processx>=3.4.4:sysutils/R-cran-processx
++TEST_DEPENDS=	R-cran-cli>0:devel/R-cran-cli \
++		R-cran-covr>0:devel/R-cran-covr \
++		R-cran-crayon>0:devel/R-cran-crayon \
++		R-cran-fansi>0:devel/R-cran-fansi \
++		R-cran-rprojroot>0:devel/R-cran-rprojroot \
++		R-cran-testthat>0:devel/R-cran-testthat \
++		R-cran-tibble>0:devel/R-cran-tibble \
++		R-cran-withr>=2.3.0:devel/R-cran-withr \
++		R-cran-ps>0:sysutils/R-cran-ps
++
++USES=		cran:auto-plist
++
++.include <bsd.port.mk>
+diff --git a/devel/R-cran-callr/distinfo b/devel/R-cran-callr/distinfo
+new file mode 100644
+index 000000000000..bc2470501953
+--- /dev/null
++++ b/devel/R-cran-callr/distinfo
+@@ -0,0 +1,3 @@
++TIMESTAMP = 1609522582
++SHA256 (callr_3.5.1.tar.gz) = ce338c648cc9ab501168a55f93e68fc81e31dc5ec881e908b5b4a9d6f5bd8696
++SIZE (callr_3.5.1.tar.gz) = 77905
+diff --git a/devel/R-cran-callr/pkg-descr b/devel/R-cran-callr/pkg-descr
+new file mode 100644
+index 000000000000..2ad375bb6b1b
+--- /dev/null
++++ b/devel/R-cran-callr/pkg-descr
+@@ -0,0 +1,5 @@
++It is sometimes useful to perform a computation in a separate R process,
++without affecting the current R process at all. This packages does exactly
++that.
++
++WWW: https://github.com/r-lib/callr#readme
```

Comment:
drop `#readme`?

Reply:
It appears that a lot of the R libraries have #readme in their WWW of R package DESCRIPTION. I feel like we could just use the original WWW in this case if it is a tradition, and it doesn't really change anything.

Reply:
That's also fine for me.

Reply:
Sounds good, thanks!